src/commands/kv/namespace/list.rs

```diff
@@ -13,19 +13,24 @@ use crate::settings::toml::Target;
 const MAX_NAMESPACES_PER_PAGE: u32 = 100;
 const PAGE_NUMBER: u32 = 1;
 
-pub fn list(target: &Target, user: &GlobalUser) -> Result<(), failure::Error> {
+pub fn print_list(target: &Target, user: &GlobalUser) -> Result<(), failure::Error> {
```

Comment:
i'd say just put  this logic in the caller (main.rs in this  case) rather than making it its own fn on this module. pulling the print out of the  original list fn is the right move though